Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed.py

```diff
@@ -326,9 +343,12 @@ def main():
         'test-module': test_module_command,
         'sixgill-get-indicators': get_indicators_command
     }
+    confidence = None
+    if "confidence" in demisto.params():
+        confidence = arg_to_number(demisto.params().get("confidence").split(" - ")[0].strip(), "confidence", required=False)
```

Comment:
```suggestion
        confidence = arg_to_number(demisto.params().get("confidence"))
```

you can remove all of this if implementing my comment in the yml and only handle the case where its `all` and not a number.